geode-assembly/src/test/resources/expected_jars.txt

```diff
@@ -37,10 +37,6 @@ jline
 jna
```

Comment:
Why is this file change in this pull request? This change is already handled by your last pull request.